sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/CosmosAuthorizationTokenResolverTest.java

```diff
@@ -51,6 +52,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Ignore("CosmosAuthorizationTokenResolver is removed from public")
```

Comment:
This is removed from public api, but still the test should work no?

Reply:
It will fail with: cannot buildAsyncClient client without any one of key, resource token, permissions.

The reason is because I removed the token resolver validation from validateConfig() in CosmosClientBuilder. I removed it because I think since the CosmosAuthorizationTokenResolver  is not public anymore, so probably also not meaningful to keep the check?

Reply:
thanks Annie for the explanation. makes sense.